metecho/api/models.py

```diff
@@ -443,7 +443,9 @@ def add_commits(self, *, commits, ref, sender):
         for epic in matching_epics:
             epic.add_commits(commits)
 
-        matching_tasks = Task.objects.filter(branch_name=ref, epic__project=self)
+        matching_tasks = Task.objects.filter(
+            Q(branch_name=ref, project=self) | Q(branch_name=ref, epic__project=self)
+        )
```

Comment:
@jerivas I don't see an obvious test that is currently exercising this, so I didn't add anything. Feel free to add a test if you'd like.

Reply:
Looks like this method is eventually called by `PushHookSerializer` which is used by `HookView` so I added a new test in 4de32e8a99c292fe827844a707864106e6e5540d